### DISCUSSION TITLE
Latest "pacman -Syu" left the system unbootable

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#1268] [Fixed Issues]
Today's (Mar 12) "pacman -Syu" left the system unbootable, even no ping possible. Here are the upgraded packages:
![grafik](https://user-images.githubusercontent.com/25383513/224533715-51b85a4b-d5ba-4219-9573-f92c27cbe032.png)
Most prominent upgrade (besides kernel and bootloader) is ffmpeg from version 5.1.2 to 6.0.

**Comment by R101G** [comment]
Confirmed, and thanks for the warning @klassikmann .

**Comment by klassikmann** [comment]
The new permissions in /srv/http/data/mpd (mpd:audio) for **all** files instead of http:http (exception mpd.db) looks suspicious. But I see @rern is just working in the UPDATE branch on that issue.

**Comment by rern** [comment]
It's something about the upgraded kernel, `linux-rpi-6.1.16-3`.
Do `pacman -Syu --ignore linux-rpi` to avoid the issue.

By the way, it happens on 32bit only.

**Reply by cmh714** [reply]
Well, my 64 bit upgrade failed. :) Figured I'd let you know.

Thank you for all you do!

**Reply by cmh714** [reply]
Also, the --ignore linux-rpi also failed to boot up for me....

**Comment by cmh714** [comment]
Ok, so since I blew up my daily image, the image I flashed is a backup from about 4 weeks ago so my pacman update is obviously more dated than klassikmann's. this is the command that so far still has me working....  pacman -Syu --ignore linux-rpi --ignore raspberrypi-bootloader

not sure that helps much....but thats whats working so far in my testing.

**Comment by R101G** [comment]
Could this be another problem which does not affect the Pi4 (like the net renaming thing)? Just in case, this is my setup:

rAudio 20230310
6.1.15-1 aarch64
Raspberry Pi 3B Rev 1.2
BCM2837 • 957 MiB
4 x Cortex-A53 @ 1.20 GHz

**Reply by cmh714** [reply]
I am same as you....

rAudio 20230310
6.1.11-2 aarch64
Raspberry Pi 3B Rev 1.2
BCM2837 • 951 MiB
4 x Cortex-A53 @ 1.20 GHz

**Comment by klassikmann** [comment]
Perhaps https://archlinuxarm.org/forum/viewtopic.php?f=67&t=16418 and https://github.com/raspberrypi/firmware/issues/1791 (pointing to the bootloader) will help to investigate the problem further.

**Comment by klassikmann** [comment]
**Workaround** (works for me):
Insert `arm_64bit=1` in `/boot/config.txt`
Valid for RPI 3B. First test (`pacman -Syu`) positive.
Found at https://github.com/raspberrypi/firmware/issues/1791#issuecomment-1465777260

**Reply by cmh714** [reply]
Excellent find! Working as you stated!

**Reply by kw-sudo** [reply]
rAudio 20230310
6.1.11-1 aarch64
Raspberry Pi 3B Rev 1.2
BCM2837 • 951 MiB
4 x Cortex-A53 @ 1.20 GHz

The workaround works for me, too! Now I'm on kernel 6.1.19-1

**Comment by klassikmann** [comment]
Additional information to the workaround:
Settings -> Backup stores (beside the data directory) also some system configuration files located in /boot, /etc, /var, among othesr also /boot/config.txt. If you apply the workaroud, boot and restore a backup e.g. from yesterday, the additional line inserted in config.txt will vanish. Therefore it is good idea to perform a backup soon after the workaround.

**Comment by R101G** [comment]
Works for me too.

**Comment by R101G** [comment]
Just to confirm that after the current update, the workaround is not needed.

**Comment by klassikmann** [comment]
**Problem solved**:
linux-rpi-6.1.19-1 and raspberrypi-bootloader-20230315-1 solve the problem. Both are part of today's `pacman -Syu`.
To disable the workaround remove the line `arm_64bit=1` from `/boot/config.txt` and make a backup in Settings -> Backup.

**Reply by cmh714** [reply]
My question is, why bother removing the workaround.....seems pretty safe and maybe better IMHO.

**Reply by klassikmann** [reply]
> My question is, why bother removing the workaround.....seems pretty safe and maybe better IMHO.

That's a matter of personal taste. I want to avoid possible (today unknown)  difficulties concerning that parameter.

**Reply by cmh714** [reply]
certainly a fair thought!!

**Comment by kw-sudo** [comment]
Yesterday I've flashed a new microDS card and did a clean install from the scratch. I could verify the following:

- image 20230212 + addon update 20230310 --> it works
- plus pacman -Syu --> After adding new music the progress bar (play) shows activity but no sound


Sytem information:


`# cat /boot/cmdline.txt
root=PARTUUID=09d2dcf4-02 rw rootwait selinux=0 plymouth.enable=0 smsc95xx.turbo_mode=N dwc_otg.lpm_enable=0 ipv6.disable=1 fsck.repair=yes isolcpus=3 console=tty3 quiet loglevel=0 logo.nologo vt.global_cursor_default=0

# cat /boot/config.txt
gpu_mem=32
initramfs initramfs-linux.img followkernel
max_usb_current=1
disable_splash=1
disable_overscan=1
dtparam=audio=off

# pacman -Qs 'firmware|bootloader' | grep ^local | cut -d/ -f2
firmware-raspberrypi 20230125-1
linux-firmware 20230210.bf4115c-1
linux-firmware-whence 20230210.bf4115c-1
raspberrypi-bootloader 20230324-1
raspberrypi-firmware 20230322-1`

**Reply by klassikmann** [reply]
@master-kw 
I think you should open a new thread for your problem because your system is bootable. And please add your system and player settings.

**Reply by kw-sudo** [reply]
[Done](https://github.com/rern/rAudio-1/discussions/1284)
